config.go

```diff
@@ -67,6 +73,13 @@ func generateSessionKey() (*ecdsa.PrivateKey, error) {
 	return key, nil
 }
 
+func (c Config) Validate() error {
+	if err := c.OIDC.Validate(); err != nil {
+		return err
```

Comment:
The OIDC.Validate() function return errors don't state anything about being OIDC, so you lose some context. Better to wrap the error here then.
```suggestion
		return fmt.Errorf("oidc config: %w", err)
```

Also, if OIDC is enabled, I believe baseURL is required? What happens if it's not set?